picoshare/guest_link.go

```diff
@@ -9,22 +9,24 @@ type (
 	GuestLinkLabel          string
 	GuestUploadMaxFileBytes *uint64
 	GuestUploadCountLimit   *int
-
-	GuestLink struct {
+	GuestUploadDisabled     *int
+	GuestLink               struct {
 		ID             GuestLinkID
 		Label          GuestLinkLabel
 		Created        time.Time
 		UrlExpires     ExpirationTime
 		FileLifetime   FileLifetime
 		MaxFileBytes   GuestUploadMaxFileBytes
 		MaxFileUploads GuestUploadCountLimit
+		Disabled       GuestUploadDisabled
```

Comment:
Can we make this `IsDisabled`? Our current convention is to use the `Is` prefix for `bool` variables.